kubecost/templates/network-costs/network-costs-daemonset.yaml

```
@@ -76,28 +76,21 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
+        - name: ELECTION_NAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
```

Thoughts on this alternative? This value is also used in Line 7 of this file.

```suggestion
          value: {{ .Release.Namespace }}
```